app/src/main/java/org/xbmc/kore/ui/sections/audio/SongsListFragment.java

```diff
@@ -192,7 +192,7 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
                 MediaProvider.Qualified.ALBUMS_THUMBNAIL
         };
 
-        String SORT = MediaDatabase.sortCommonTokens(MediaProvider.Qualified.SONGS_TITLE) + " ASC";
+        String SORT = MediaDatabase.sortCommonTokens(MediaProvider.Qualified.SONGS_TITLE) + " COLLATE NOCASE ASC";
```

Comment:
Over here it might have some impact on performance, in big libraries. Let's keep it, but if someone complains we'll revisit this.